### PR TITLE
Add support for Starlette's GZip middleware

### LINKED ISSUE #45: Add support for Starlette's GZip middleware
**Is your feature request related to a problem? Please describe.**
Starlette's GZip middleware is really useful for big responses and easy to add.

**Describe the solution you'd like**
Add a flag that should enable GZip middleware on API initialization.

### PR DESCRIPTION
**Pull request by alin23**
Closes #45

### CODE REVIEW

File: docs/topics/features/gzip.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# GZip
+
+If you want to enable [GZip compression](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding#Directives) compress responses when possible, simply use:
```
Comment (florimondmanca) [inline docs/topics/features/gzip.md]: I think 'to' is missing here :)
```suggestion
If you want to enable [GZip compression](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding#Directives) to compress responses when possible, simply use:
```

File: docs/topics/features/gzip.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+```python
+api = bocadillo.API(enable_gzip=True)
+```
+
+You can also specify the minimum bytes the response should have before compressing:
+```python
```
Comment (florimondmanca) [inline docs/topics/features/gzip.md]: Missing newline for consistency:

```suggestion

```python
```

File: tests/test_gzip.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from bocadillo import API
+
+
+def test_if_gzip_enabled_and_response_is_compressed():
```
Comment (florimondmanca) [inline tests/test_gzip.py]: I think you meant 'then' here:

```suggestion
def test_if_gzip_enabled_then_response_is_compressed():
```

**Comment by alin23** [comment]
Good points! I commited all your fixes

**Pull request merged**